libs/solid/profile/src/lib/components/base/base.component.ts

```diff
@@ -15,6 +16,10 @@ import { map } from 'rxjs/operators';
 import { BehaviorSubject, combineLatest, Observable } from 'rxjs';
 import { Dispatch } from '@ngxs-labs/dispatch-decorator';
 import { ProfileActions } from '../../state/profile.actions';
+import {
+  SOLID_SKELETON_CONFIG,
+  SolidSkeletonConfig,
+} from '@zentrumnawi/solid-skeleton';
 
```

Comment:
Ich konnte nicht von solid-skeleton in component solif-profile importieren, da es hier Lint problem gibt. Ich brauche diese Importierung dadurch kann ich die custom url zugreifen. Man kann nur von solid-core importieren oder? Dann weiß ich nicht wie ich es konfigurieren kann. Vielleicht hast du @sheepyhollow eine Idee.

Reply:
Hast Du verstanden, was Linting-Fehler erzeugt? 

Mein best guess ist, dass es hier passiert:
https://github.com/zentrumnawi/solid-frontend/blob/2a851330870c627d47dbdc674ac9f6525d33428a/libs/solid/skeleton/src/lib/solid-skeleton-config.ts#L106

Ich verstehe außerdem nicht, warum `SOLID_SKELETON_CONFIG` und  `SolidSkeletonConfig` zusammen da stehen, aber das ist eine andere Sache, denke ich.

Reply:
Lint sagt, dass die dependencies zwischen skeleton und profile nicht erlaubt sind. Was ich hier selbe hinzugefügt habe macht ein Custom Config für die url anstatt _profile_ wie vorher (Zeile 176). Also wenn ich von _solid-core_ importiere ist es kein Problem aber von _solid-skeleton_ gibts dependency problem bei Lint.

Reply:
Ich probier mal was

Reply:
Das Problem ist (denke ich), dass `solid-profile` in `solid-skeleton-config.ts` importiert wird (siehe oben). Damit entsteht eine _circular dependency_.

Reply:
> Ich verstehe außerdem nicht, warum `SOLID_SKELETON_CONFIG` und `SolidSkeletonConfig` zusammen da stehen, aber das ist eine andere Sache, denke ich.

`SolidSkeletonConfig` ist das Interface für die Config. Und `SOLID_SKELETON_CONFIG` ist der Injection Token, den Angular braucht um das Objekt zuzuordne.
Hier die entsprechende Doku: [https://angular.io/guide/dependency-injection-providers#injecting-an-object](url)